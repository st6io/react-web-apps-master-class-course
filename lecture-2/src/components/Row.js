import React from 'react';

import styles from './common.module.scss';

const Row = ({ children }) => <div className={styles.row}>{children}</div>;

export default Row;
