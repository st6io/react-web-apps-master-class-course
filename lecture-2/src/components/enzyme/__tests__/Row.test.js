import React from 'react';
import { shallow } from 'enzyme';

import Row from '../../Row';

describe('Row Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <Row className="new-class-name">
        <span>Hello, World!</span>
      </Row>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
