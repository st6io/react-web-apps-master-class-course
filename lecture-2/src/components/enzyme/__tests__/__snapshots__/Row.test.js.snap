// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Component should match snapshot 1`] = `
<div
  className="row"
>
  <span>
    Hello, World!
  </span>
</div>
`;
